platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/ImportMigrationEntry.java

```diff
@@ -185,6 +198,10 @@ public default boolean restore() {
    *     <code>false</code> otherwise
    * @throws MigrationException if a failure that prevents the operation from continuing occurred
    * @throws IllegalArgumentException if <code>consumer</code> is <code>null</code>
+   * @throws SecurityException if a security manager exists and the entry was exported by the
+   *     framework using {@link ExportMigrationEntry#store()} or {@link
```

Comment:
❓ Wouldn't restore need write access?

Reply:
This restore is not writing to disk, it is accessing the stream of something exported. If that something was exported by the framework and not by the migratable than it better have read permissions to get its hands on.